shared/wallets/transaction-details/index.js

```diff
@@ -359,6 +350,18 @@ const TransactionDetails = (props: NotLoadingProps) => {
             </Kb.Text>
           )}
         </Kb.Box2>
+        <Kb.Box2 direction="vertical" gap="xxtiny" fullWidth={true} style={styles.buttonBox}>
+          {props.onCancelPayment && (
```

Comment:
Could you move this up a line to cover the `Box2` as well?

Reply:
ok